### ISSUE TITLE
Health Checks (NEST-513)

### ISSUE THREAD

**Issue by hishamco** [Lombiq/DotNest-Support#53]
I did some work related to the Health Checks feature by extending what OC offers to us, the Health Check UI was super cool for DotNest and might need some effort and enhancements. I remember Sebastien mentioned that a long time ago during the meeting when I demo the feature

Is there a plan to include such a feature?

[Jira issue](https://lombiq.atlassian.net/browse/NEST-513)

**Comment by Piedone** [comment]
[Here's](https://www.youtube.com/watch?v=GeV_n7e4Niw) the mentioned demo. While this is great, I don't think DotNest needs something like this yet. Eventually, perhaps yes, but then with an external status page that you can also access if DotNest itself is down.

**Comment by hishamco** [comment]
I think the health checks are useful in many cases: for the management similar to https://www.githubstatus.com/ also, for the user to check the tenant services statuses

**Comment by Piedone** [comment]
Yeah, I was talking about something like https://www.githubstatus.com/. This we'll definitely need eventually, just not really necessary yet.

**Comment by hishamco** [comment]
We might add one for Orchard Core it will be useful for each site
